two_factor/views/core.py

```diff
@@ -213,7 +213,10 @@ class SetupView(IdempotentSessionWizardView):
         ('validation', DeviceValidationForm),
         ('yubikey', YubiKeyDeviceForm),
     )
+
+    show_welcome = not hasattr(settings, 'TWO_FACTOR_SKIP_WELCOME') or not settings.TWO_FACTOR_SKIP_WELCOME
```

Comment:
This will be executed at import time, not when the lambda function in the `condition_dict` is being called. Can you either define the lambda expression here or move this into the lambda function's body, please.